third_party/blake3/blake3.BUILD

```diff
@@ -1,102 +0,0 @@
-load("@rules_license//rules:license.bzl", "license")
```

Comment:
Should I split this out?

Reply:
I think it's fine to migrate to BCR in one PR. However, can you create another PR for the benchmark code? Thanks!

Reply:
This is not the migration, this file was unused even before this PR. I thought that I might as well remove it, but I remember `third_party` changes being subject to a special merge procedure.

I removed this from the diff and will submit as a separate PR.

Reply:
https://github.com/bazelbuild/bazel/pull/22032